docs/features/feature-guides/properties/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Structured Properties
+<FeatureAvailability/>
+
+**DataHub Structured Properties** provide a structured and validated approach to attaching metadata to your Data Assets, enabling organizations to enforce consistency and quality in metadata collection.
```

Comment:
I'd remove DataHub from the bold here.

The feature is "Structured Properties" inside the world of DataHub, so only that part should be bold